src/main/java/org/opensearch/security/identity/SecurityTokenManager.java

```diff
@@ -0,0 +1,261 @@
+package org.opensearch.security.identity;
```

Comment:
The SecurityTokenManager should use an instance of a JWTVendor to issue JWTs. There are now multiple places where this repo issues JWTs and in each instance they would use a different JWK (signing key) when signing the JWT. 

1. OBO and Service Account Tokens for extensions would use a JWTVendor to issue a JWT.
2. The SAML auth flow could be refactored to utilize a JWTVendor.

Reply:
Peter's request: https://github.com/opensearch-project/security/pull/3538#discussion_r1362149896

Reply:
SecurityTokenManager uses an instance of JwtVendor again